src/Features/Navigation/CoreMenu.php

```diff
@@ -216,7 +216,7 @@ public static function get_items() {
 					'id'         => 'woocommerce-marketplace',
 					'url'        => 'wc-addons',
 					'menuId'     => 'secondary',
-					'order'      => 20,
+					'order'      => 10,
```

Comment:
Nice 👍 

Let's also update the `Settings` order to `20` (should be around line 107) in case we need to add things via extensible filters/hooks later.

Reply:
updated!